### ISSUE TITLE
missing code dependency

### ISSUE THREAD

**Issue by freedom-foundation** [seSuperuser/Superuser#24]
MagiskSU relies on selinux code not included.

**Comment by lbdroid** [comment]
This is not magisk

**Comment by freedom-foundation** [comment]
> This is not magisk

MagiskSU is magisk, yes? How are you supposed to build it? So the magisk app is supposed to call su on a hidden partition but you thing MagiskSU is not part of magisk? It is actually the main function with little appie junk built around it. I don't know if I would trust a programmer this stupid. Can you move out of the way and put John Wu on the line?

**Comment by freedom-foundation** [comment]
oh and if you mean the repo magisk. I posted here because MagiskSU is a clone of this same missing dep would apply.

**Comment by freedom-foundation** [comment]
let me rephrase: Superuser relies on selinux code not included.
